rialto_airflow/dags/harvest.py

```diff
@@ -164,33 +164,34 @@ def link_funders(snapshot, openalex_fill_in, dimensions_fill_in):
     def publish_openaccess(snapshot):
         openaccess.write_publications(snapshot)
         openaccess.write_contributions(snapshot)
+        openaccess.write_contributions_by_school(snapshot)
+        openaccess.write_contributions_by_department(snapshot)
 
     @task()
     def upload_publish_files(snapshot):
```

Comment:
Instead of repeating this for the additional files I figured it would be easier to loop over them.